src/app/components/Application/Main.jsx

```diff
@@ -1,33 +1,45 @@
+/* globals document */
 import React from 'react';
-import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 
 import BookList from '../BookList/BookList';
 import Sidebar from '../Sidebar/Sidebar';
 import utils from '../../utils/utils';
+import staffPicksDate from '../../utils/DateService';
+import appConfig from '../../../../appConfig';
 
 class Main extends React.Component {
   constructor(props) {
     super(props);
 
+    const picksData = this.props.picksData;
+    const picks =
```

Comment:
The picks always need to be filtered when they are set initially by the age type. This will not be the case for `childrens` or `ya` lists the function covers that.